.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "pip" # See documentation for possible values
+    directory: "/docs/.sphinx" # Location of package manifests
```

Comment:
Will this limit dependabot to just checking docs/.sphinx directory?

Reply:
I think we want it check everything.

Reply:
I have added the following requirements.txt files in [b3ecfae](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/pull/1727/commits/b3ecfae1f71d21751c4777ee22992f6385f7d0e4)

```
./tools/accuracy/requirements.txt
./docs/.sphinx/requirements.txt
./examples/vision/python_unet/requirements.txt
./examples/vision/python_super_resolution/requirements.txt
./requirements.txt
```